cmd/agent/common/misconfig/mounts.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package misconfig
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func procMount() error {
+	groups, err := os.Getgroups()
+	if err != nil {
+		return fmt.Errorf("failed to get process groups: %v", err)
+	}
+	return checkProcMountHidePid("/host/proc/1/mounts", groups)
```

Comment:
```suggestion
	return checkProcMountHidePid(filepath.Join(config.Datadog.GetString("container_proc_root"), "1/mounts"), groups)
```

I’m wondering if we should mimic [this logic (found in `integrations-core`)](https://github.com/DataDog/integrations-core/blob/22dd4c4015cdd30aa7a778076683d1b01b90b862/network/datadog_checks/network/network.py#L489-L493) where the `/1/` part is added only in case we have an evidence that the agent is containerised.